eng/Versions.props

```diff
@@ -9,19 +9,16 @@
     <AspNetCoreMajorVersion>3</AspNetCoreMajorVersion>
     <AspNetCoreMinorVersion>2</AspNetCoreMinorVersion>
     <AspNetCorePatchVersion>0</AspNetCorePatchVersion>
-    <PreReleasePreviewNumber>5</PreReleasePreviewNumber>
+    <PreReleasePreviewNumber>1</PreReleasePreviewNumber>
```

Comment:
Why not use the Arcade SDK's `$(PreReleaseVersionIteration)` variable? It will then do the **Right Thing:tm:** for both semver1 and 2.

Reply:
Given that we're very close to shipping Blazor WASM, I would prefer to keep things simple and stick to the approaches we've used past several months, to avoid any potential disruptions.